File: 프로그래머스/1/12926. 시저 암호/시저 암호.js
function solution(s, n) {
    var answer = '';
    
    // 반복문을 통해 s 문자열을 순회하며
    for (i = 0; i < s.length; i++) {
        // 만약 문자의 아스키코드가 소문자라면
        if (s[i].charCodeAt() >= 97 && s[i].charCodeAt() <= 122) {
            answer += String.fromCharCode((s[i].charCodeAt() - 97 + n) % 26 + 97);
        } else if (s[i].charCodeAt() >= 65 && s[i].charCodeAt() <= 90) { // 대문자라면
            answer += String.fromCharCode((s[i].charCodeAt() - 65 + n) % 26 + 65);
        } else {
            answer += s[i];
        }
    }
    
    return answer;
}

File: 프로그래머스/1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    answer = ''
    
    for i in s:
        if i.islower(): # 소문자일 경우
            answer += chr((ord(i) - ord('a') + n) % 26 + ord('a'))
        elif i.isupper(): # 대문자일 경우
            answer += chr((ord(i) - ord('A') + n) % 26 + ord('A'))
        else: # 대소문자가 아닐 경우 i를 그대로 추가
            answer += i
    
    return answer